File: codigos.js.py/js_html/7c.js
/*Classe Bola: Crie uma classe que modele uma bola:

    Atributos: Cor, circunferência, material
    Métodos: trocaCor e mostraCor */

var bola = {
  cor: "alvinegra",
  circunferencia: 25,
  material: "coro",

  trocarCor: function(nCor){
    this.cor = nCor;
  },

  mostarCor: function(){
    return this.cor;

  }

};

document.write("Cor original da bola = ", bola.mostarCor());
bola.trocarCor("verde");
document.write(" => Nova Cor  = ", bola.mostarCor());

File: codigos.js.py/scripts_py/7c.py
class Bola:
    def __init__(self, cor, circunferencia, material):
        self.cor = cor
        self.circunferencia = circunferencia
        self.material = material

    def troca_cor(self, n_cor):
        self.cor = n_cor

    def mostrar_cor(self):
        return self.cor

bola =  Bola('preta', 25,'couro')
print(bola.mostrar_cor())
bola.troca_cor('verde')
print(bola.mostrar_cor())
